orchestrator_mapreduce.py

```
@@ -73,6 +87,483 @@ class OrchestratedJob:
+                "layer": "mechanical",
+                "success_rate": rate,
+                "reason": (
+                    f"Success rate {rate:.0%} below "
+                    f"{self.threshold:.0%} threshold"
+                ),
+            }
+
+        # Layer 2: Semantic evaluation via LLM
+        try:
+            semantic_result = await self._llm_evaluate(job)
+            return semantic_result
```

Ruff (0.15.18)

[warning] Consider moving this statement to an `else` block

(TRY300)